sdcm/cluster.py

```diff
@@ -922,7 +922,7 @@ def start_task_threads(self):
             self.start_journal_thread()
             self.start_backtrace_thread()
             self.start_db_log_reader_thread()
-        if 'monitor' in self.name:
+        if 'monitor' in self.name and Setup.DECODING_QUEUE:
```

Comment:
there is an SCT param `backtrace_decoding` that controls if it is enabled

Reply:
use `Setup.BACKTRACE_DECODING`

Reply:
@bentsi i thought about it, but but use DECODING_QUEUE to be sure that queue thread is started

Reply:
if the thread starts and `Setup.BACKTRACE_DECODING` is `false` we have a bug that we should know about.
(the thread should start only when we want it to, and the way to control it is by setting `backtrace_decoding` flag and nothing else)